packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -4,17 +4,16 @@ import {
     Modal,
     Tabs,
     Button,
-    DesktopWrapper,
     Div100vhContainer,
-    MobileWrapper,
     MultiStep,
     PageOverlay,
     ThemedScrollbars,
     UILoader,
     Text,
 } from '@deriv/components';
+import { useDevice } from '@deriv-com/ui';
 import { localize, Localize } from '@deriv/translations';
-import { isMobile, getCFDPlatformLabel } from '@deriv/shared';
+import { isMobile, getCFDPlatformLabel, isDesktop } from '@deriv/shared';
```

Comment:
IsDesktop and IsMobile from Shared package are deprecated. you need to use the one from `UseDevice` in deriv-com/ui.

Reply:
Resolved, it was auto imported. Replaced it with the one from useDevices.